Libplanet/Blockchain/BlockChain.cs

```diff
@@ -624,7 +639,7 @@ public void StageTransaction(Transaction<T> transaction)
                 if (!_transactions.ContainsKey(transaction.Id))
                 {
                     _transactions[transaction.Id] = transaction;
```

Comment:
Even if we manage the stage in-memory, it will still use a disk if add the `transaction` to the store here.

Reply:
Applied: a496a0edb6785417491b56088f25fe5ac652e6cd.